modules/widgets/facebook-likebox.php

```diff
@@ -42,6 +37,8 @@ function widget( $args, $instance ) {
 
 		$like_args = $this->normalize_facebook_args( $instance['like_args'] );
 
+		wp_enqueue_style( 'jetpack_facebook_likebox', plugins_url( 'facebook-likebox/style.css', __FILE__ ) );
```

Comment:
Alternately, we can just roll it into our concatenated stylesheet and load it in there.  Either way.